pom.xml

```diff
@@ -277,6 +277,7 @@
         <configuration>
           <ignoredDependencies combine.children="append">
             <ignoredDependency>com.typesafe.play:play-java_${scala.package.version}:jar:*</ignoredDependency>
+            <ignoredDependency>net.markenwerk:utils-mail-dkim:jar:1.1.12</ignoredDependency>
```

Comment:
So I suspect the reason you are doing this is because there are no non-test references to this package. Why not simply make the direct dependency on this package scoped to `test` (and remove the exception here)?